### ISSUE TITLE
[DRAFT] Save and Subscribe to a Search - Not Ready for Feedback

### ISSUE THREAD

**Issue by mxk0** [HHS/simpler-grants-gov#2820]
# Not ready for feedback

### Summary 

Today, users must visit simpler.grants.gov and enter a search to see the latest results for a particular keyword/filter combination. This results in a lot of manual repetitive effort, and may cause grant applicants to miss key opportunities if they're on vacation, forget to run their weekly search, etc. By allowing users to save and subscribe to searches, we can save them time and ensure they learn about relevant opportunities as soon as they're published.

This user need and a proposed solution [came out of the Collaborative Coding and Design Challenge](https://docs.google.com/presentation/d/1BAhUlzwG6ET7DSWJHNL3JLGMC5b9j29K-GqIv_1Hr_U/edit#slide=id.p).

### Press release

Today, Simpler.Grants.gov is launching an exciting new time saving feature for grant seekers: the ability to save a search and receive email updates when new opportunities meeting their search criteria are published. Until now, users visited the site regularly to check for new opportunities relevant to their needs. With this new feature, users can spend more time on high-value work for their organization and less time refreshing a search page.

### Acceptance criteria

- [ ] Users can save a search
- [ ] Users can view their saved searches
- [ ] Users can open a saved search
- [ ] Users can delete a saved search
- [ ] Users can subscribe to a saved search, receiving email notifications about any newly announced opportunities meeting the saved search keywords/filters
- [ ] Users can unsubscribe from a saved search in one click from an email notification
- [ ] Users can click through to individual opportunities or to the saved search from an email notification

### Metrics

- Set a baseline for number of users who use the saved search feature
- Set a baseline for number of saved searches per user who uses the feature
- Set a baseline for number of users who create subscriptions
- Set a baseline for number of subscriptions per user who uses the feature
- Opportunity listing page views (expected to increase due to email notifications)

### Assumptions

This spec assumes that both saving searches and subscribing to searches deliver user value. As a method of iterative development, we will first deliver the ability to subscribe to a search without the saved search feature, as laid out in the [CC&DC presentation](https://docs.google.com/presentation/d/1BAhUlzwG6ET7DSWJHNL3JLGMC5b9j29K-GqIv_1Hr_U/edit#slide=id.p). That functionality will then be absorbed into the saved search feature once saved search is implemented.

### Dependencies

- This feature depends on #2640

### Open questions

- Should the user receive an email per new matching opportunity? Should we do any batching? If so, batch within a saved search, or batch results from multiple saved searches within the same email?

**Comment by mxk0** [comment]
Closing in favor of #2875
